lib/openhab/dsl/items/builder.rb

```diff
@@ -187,6 +187,30 @@ class << self
           def item_factory
             @item_factory ||= org.openhab.core.library.CoreItemFactory.new
           end
+
+          #
+          # Convert the given array to an array of strings.
+          # Convert Semantics classes to their simple name.
+          #
+          # @param [String,Symbol,Semantics::Tag] tags A list of strings, symbols, or Semantics classes
+          # @return [Array] An array of strings
+          #
+          # @example
+          #   normalize_tags("tag1", Semantics::LivingRoom)
+          #
+          # @!visibility private
+          def normalize_tags(*tags)
```

Comment:
@ccutrer is there a better place to put this method in? It's used by GenericItem#tags, Item builder, and automation_rule for rule tags.

Reply:
:shrug:. Maybe a new TagHelper module or something, but this is fine for now.